docs/topics/using/edgectl/service-preview-install.md

```diff
@@ -54,7 +243,202 @@ See the [Traffic Manager reference](../service-preview-reference#traffic-manager
 
 The traffic manager is now installed in the Ambassador namespace in your cluster and is ready to connect your cluster to your local machine.
 
-### 2. Connect to your Cluster
+### 4. Connect to your Cluster
```

Comment:
Do we really want to have duplicates of this `Connect to your Cluster` section?

Reply:
This was the thing I was trying to simplify. 

Technically, the installation of the `Traffic Manager` and `Ambassador Injector` is the only change. The docs currently have `## Install with Edgectl` and `## Install with YAML` sections but the `Traffic Agent` installation is left only for the `Install with YAML` section.

My goal was to have three complete install sections that run through the full installation of SP with each method. My opinion is that this is much clearer than saying:

> ## Install the Traffic Manager and Ambassador Injector
> 
> The Traffic Manager is what is responsible for managing communications between your Kubernetes cluster and your local machine.
> 
> Services in your cluster opt-in to using Service Preview by injecting the Traffic Agent sidecar. Service Preview includes an automatic sidecar injection feature which simplifies the process of injecting the Traffic Agent as sidecars to your services.
> 
> Select your installation method below to learn how to install the Traffic Manager and Ambassador Injector
> 
> ### Edgectl
> If you installed Ambassador with `edgectl install`, then the `Traffic Manager` and `Ambassador Injector` are already installed
> 
> ### YAML
> 
> Deploy the Traffic Manager and Ambassador Injector in the `ambassador` namespace with `kubectl`:
> 
> ```
> kubectl apply -f https://getambassador.io/yaml/traffic-manager.yaml
> kubectl apply -f https://getambassador.io/yaml/ambassador-injector.yaml
> ```
> 
> ...
> 
> ### Helm
> 
> Install Service Preview alongside the Ambassador Edge Stack with the following `values.yaml` options:
>
> ```yaml
> servicePreview:
>   enabled: true
> ```
>
> Create the Ambassador namespace if it is not already created:
>
> ```sh
> $ kubectl create namespace ambassador
> ```
>
> Upgrade or install your release of the Ambassador Edge Stack with the Traffic Manager and Ambassador Injector
>
> ```sh
> $ helm upgrade --install ambassador -n ambassador datawire/ambassador -f values.yaml
> ```
>
> ## 2. Install the Traffic Agent

I am open to changing this but that was my approach